addons/email_template/migrations/8.0.1.1/pre-migration.py

```diff
@@ -24,6 +24,9 @@
 column_renames = {
     'email_template': [
         ('email_recipients', 'partner_to'),
+    ],
+    'ir_act_server': [
+        ('subject', 'old_subject'),
```

Comment:
Use None here, and refer to the old subject column name in the post script with openupgrade.get_legacy_name('old_subject')